### PR TITLE
Update rule CCE-83441-6 with RHEL9 STIG assessment

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Update the rule with SRG assessment info
- The text about strict FIPS compliance has been moved from the `fixtext` to the `description`.

#### Rationale:

- Prepare the rule for RHEL9 STIG

### CODE REVIEW

File: linux_os/guide/system/software/integrity/fips/sysctl_crypto_fips_enabled/rule.yml [hunk 1]
```diff
@@ -61,17 +65,39 @@ warnings:
...
 srg_requirement:
-    '{{{ full_name }}} must implement NIST FIPS-validated cryptography for the following: to provision digital signatures, to generate cryptographic hashes, and to protect data requiring data-at-rest protections in accordance with applicable federal laws, Executive Orders, directives, policies, regulations, and standards.'
+    '{{{ full_name }}} must implement NIST FIPS-validated cryptography for the following: to
+    provision digital signatures, to generate cryptographic hashes, and to
+    protect data requiring data-at-rest protections in accordance with
+    applicable federal laws, Executive Orders, directives, policies,
+    regulations, and standards.'
+
+vuldiscussion: |-
+    If maintenance tools are used by unauthorized personnel, they may accidentally or intentionally
```
Comment (Mab879) [inline linux_os/guide/system/software/integrity/fips/sysctl_crypto_fips_enabled/rule.yml:82]: Please remove this does not match the Vul Discussion in the spreadsheet.

**Review by Mab879 — CHANGES_REQUESTED** [review]

File: linux_os/guide/system/software/integrity/fips/sysctl_crypto_fips_enabled/rule.yml (hunk 1 above)
Comment (yuumasato) [reply linux_os/guide/system/software/integrity/fips/sysctl_crypto_fips_enabled/rule.yml:82]: Hmm, it does match the spreadsheet, but `SRG VulDiscussion`, not `Vul Discussion`.

Now I'm confused, does the `SRG VulDiscussion` matter?

File: linux_os/guide/system/software/integrity/fips/sysctl_crypto_fips_enabled/rule.yml [hunk 2]
```diff
@@ -61,17 +65,39 @@ warnings:
...
 
 fixtext: |-
-    Configure {{{ full_name }}} to implement DoD-approved encryption by following the steps below:
-
-    To enable strict FIPS compliance, the fips=1 kernel option needs to be added to the kernel boot
-    parameters during system installation so key generation is done with FIPS-approved algorithms
-    and continuous monitoring tests in place.
-
-    Enable FIPS mode after installation (not strict FIPS compliant) with the following command:
+    Configure the operating system to implement FIPS mode with the following command
```
[the remark this answers is not in the document]
Comment (yuumasato) [reply linux_os/guide/system/software/integrity/fips/sysctl_crypto_fips_enabled/rule.yml:68]: Well, the spreadsheet didn't have the `:`, :stuck_out_tongue: .

[the remark this answers is not in the document]
Comment (yuumasato) [reply linux_os/guide/system/software/integrity/fips/sysctl_crypto_fips_enabled/rule.yml]: Fixed by rebase.

File: linux_os/guide/system/software/integrity/fips/sysctl_crypto_fips_enabled/rule.yml (hunk 1 above)
Comment (Mab879) [reply linux_os/guide/system/software/integrity/fips/sysctl_crypto_fips_enabled/rule.yml:82]: The `SRG VulDiscussion` is a static field used during the development of a STIG. We want the `Vul Discussion` which is specific to our product.

**Pull request merged**